staging/src/k8s.io/kube-aggregator/OWNERS

```diff
@@ -18,5 +18,6 @@ reviewers:
   - jpbetz
 labels:
   - sig/api-machinery
+  - sig/etcd
```

Comment:
also api-machinery.

Reply:
moved this to `staging/src/k8s.io/kube-aggregator/pkg/registry/apiservice/etcd`

Reply:
will wait for sig-apimachinery review